### PR TITLE
[v7r0] RSS: Backported GGUSTicketsPolicy from LHCb

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*ResourceStatusSystem
NEW: Backported GGUSTicketsPolicy from LHCb
CHANGE: removed dangerous script

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Please review, and ignore the failing test for now.

File: Core/LCG/GGUSTicketsClient.py [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
 __RCSID__ = "$Id$"
 
-class GGUSTicketsClient:
 
-  # create the URL to get tickets relative to the site ( opened only ! ): 
+def getGGUSURL(vo=None, siteName=None):
+  """ create the URL to get tickets relative to the site ( opened only ! ):
+  """
+  if vo is None:
+    gConfig.getOption('DIRAC/VirtualOrganization')
```
Comment (atsareg) [inline Core/LCG/GGUSTicketsClient.py]: Did you forget to assign the result of the getOption() call ?

Comment (fstagni) [reply Core/LCG/GGUSTicketsClient.py]: Thanks, fixed.

**Comment by atsareg** [comment]
@andresailer : Approved ?

**Pull request merged**